src/wire/protocol.cpp

```diff
@@ -153,10 +149,8 @@ void PacketManager::PutTupleDescriptor(
 void PacketManager::SendDataRows(std::vector<ResultType> &results, int colcount,
                                  int &rows_affected,
                                  ResponseBuffer &responses) {
-  LOG_TRACE("result size: %ld, colcount: %d", results.size(), colcount);
   if (!results.size() || !colcount) return;
```

Comment:
Can you changes this to : `results.empty() || col_count == 0`. Try not to use `!`. Extra effort for the reader.